src/plots/geo/constants.js

```diff
@@ -24,24 +24,22 @@ exports.projNames = {
     'winkel tripel': 'winkel3',
     'aitoff': 'aitoff',
     'sinusoidal': 'sinusoidal',
-/*
-    // potential projections that could be added to the API
 
     'airy': 'airy',
     // 'albers': 'albers',
-    'armadillo': 'armadillo',
+    // 'armadillo': 'armadillo',
```

Comment:
Is there a pattern to the ones we're still omitting?

Reply:
Some of these are now enabled by https://github.com/plotly/plotly.js/pull/5801/commits/eb30073d2e0f23093fba33324691457f94319c44.
Let me see if I could easily fix the clipping issue for some other remaining ones...

Reply:
Most of the remaining ones are [interrupted-projections ](https://github.com/d3/d3-geo-projection#interrupted-projections) and [polyhedral-projections](https://github.com/d3/d3-geo-projection#polyhedral-projections) as well as [two-point projections ](https://github.com/d3/d3-geo-projection#geoTwoPointAzimuthal), require extra work and attributes and we could say it is outside the scope of this PR.